### DISCUSSION TITLE
Avoid upgrading at the same time

### DISCUSSION THREAD

**Discussion by wangxinxi** [containrrr/watchtower#1329] [Q&A]
My web app has redundancy for high availability. How shall I configure watchtower to avoid upgrading the docker containers simultaneously, which will shut the entire site down? 

Is there any solution to configure a delay for each replica so that the upgrading won't happen at the same time?

**Comment by simonaronsson** [comment]
You'd have to have multiple watchtowers running, and scope them to one deployment each with a staggered schedule.
